### DISCUSSION TITLE
update Marker to AdvancedMarkerElement

### DISCUSSION THREAD

**Discussion by pkitas17** [inocan-group/vue3-google-map#201] [General]
Hello, I have a problem with your dependency, since February 21st google asks us to use google.maps.marker.AdvancedMarkerElement instead of google.maps.Marker, which causes a warning in the console every time markers are loaded in the map. I checked your documentation to see if you can implement advanced marker element directly, but I didn't find a way.

**Comment by HusamElbashir** [comment]
You can implement it yourself if you wanted to through the native API. See:
https://github.com/inocan-group/vue3-google-map?tab=readme-ov-file#advanced-usage

We do intend to implement this component at some point. You can subscribe to this issue if you want to be notified when we release that feature:
https://github.com/inocan-group/vue3-google-map/issues/199
